packages/grid/src/GridTypes.ts

```diff
@@ -0,0 +1,498 @@
+import GridModel from './GridModel';
+import GridRange from './GridRange';
```

Comment:
This is actually creating a circular import right now; GridRange imports GridTypes, and GridTypes imports GridRange. Likewise, GridModel imports GridTypes and vice versa.
What was the original circular import we were trying to avoid with this change? I think need to provide a bit more guidance here on what cleanup is needed

Reply:
We were originally trying to avoid `CellRenderer` importing `GridRenderState` from `GridRenderer`, which would also import `CellRenderer`.

Reply:
Right, see my comment: https://github.com/deephaven/web-client-ui/pull/1212#discussion_r1166971643
I think that's really all that's needed